### PR TITLE
Create top up account screen and fix sdk initiation

### PR DESCRIPTION
**Pull request by MouazAlzahabi**
Handles #3222 #3166 

Changes proposed in this pull request:
- Create select top up screen logic only, still need to do some UI adjustment later
- Create signature and fix token filtering issue

### CODE REVIEW

File: Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift [hunk 1]
```diff
@@ -96,7 +96,17 @@ class BalancesViewController: LoadableViewController, UITableViewDelegate, UITab
             guard let safe = try? Safe.getSelected() else {
                 return
             }
-            App.shared.ramper.startOnRamp(safe: safe)
+
+            let vc = SelectTopUpAddressViewController()
```
Comment (biafra23) [inline Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift:100]: Does it make sense to de-duplicate this? Looks like it is the same as in AssetsViewController.

File: Jenkinsfile [hunk 2]
```diff
@@ -24,6 +24,8 @@ pipeline {
         INTERCOM_PROD_APP_ID = credentials('INTERCOM_PROD_APP_ID')
         MOONPAY_STAGING_API_KEY = credentials('MOONPAY_STAGING_API_KEY')
         MOONPAY_PROD_API_KEY = credentials('MOONPAY_PROD_API_KEY')
+	MOONPAY_STAGING_SECRET_KEY = credentials('MOONPAY_STAGING_SECRET_KEY')
```
Comment (elgatovital) [inline Jenkinsfile:27]: what is a secret key? What's the difference btw api and secret key?

File: Multisig/UI/Assets/Ramper.swift [hunk 3]
```diff
@@ -9,59 +9,69 @@
...
 
 class Ramper: MoonpayCallbackInterface {
 
     private let MOONPAY_THEME_ID_SAFE = "7e8968f6-99a4-43b5-8286-d3c290d3a0b2"
-    private let moonpay: MoonpaySDK
+    private let moonpay: MoonPayiOSSdk
 
     init() {
-        moonpay = MoonpaySDK()
+        moonpay = MoonPayiOSSdk()
```
Comment (elgatovital) [inline Multisig/UI/Assets/Ramper.swift:20]: MoonPayiOSSdk is hard to read. Maybe we should give moonpay team feedback about naming :)

File: Jenkinsfile (hunk 2 above)
Comment (elgatovital) [reply Jenkinsfile:27]: probably this https://docs.moonpay.com/moonpay/developer-resources/api/server-side-apis/authentication

Comment (elgatovital) [reply Jenkinsfile:27]: I don't understand why we need both

**Review by elgatovital — APPROVED** [review]

File: Multisig/UI/Assets/Ramper.swift (hunk 3 above)
Comment (MouazAlzahabi) [reply Multisig/UI/Assets/Ramper.swift:20]: This is not the only thing I would like to report actually, but I won't be in favor of waiting for another version of the sdk

File: Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift (hunk 1 above)
Comment (MouazAlzahabi) [reply Multisig/UI/Assets/BalancesViewController/BalancesViewController.swift:100]: Will do in the next pr

**Pull request merged**